README.adoc

```diff
@@ -94,6 +119,21 @@ In your YAML file, add:
           language: LANGUAGE_PYTHON
 ----
 
+In the following example, if there are any critical, very high or high severity vulnerabilities identified, the build is broken.
+
+[source,yaml]
+----
+      - name: Scan with SonarQube bundled
+        uses: cloudbees-io/sonarqube-bundled-sast-scan-code@v2
+        with:
+          sonar-exclusion: sonar/*
+          cover-file-name: ./coverage.xml
+          language: LANGUAGE_PYTHON
+          threshold-critical: 0
+          threshold-very-high: 0
+          threshold-high: 0
```

Comment:
Can you add medium and low in here too ?

Reply:
added in https://github.com/cloudbees-io/sonarqube-bundled-sast-scan-code/pull/26/commits/35de965ea1143833b79ca7bc696361308cdce75a